Simplenote/AccountVerificationController.swift

```diff
@@ -71,7 +71,7 @@ class AccountVerificationController: NSObject {
 
     /// Send verification request
     ///
-    func verify(completion: @escaping (_ result: Result<Int, RemoteError>) -> Void) {
+    func verify(completion: @escaping (_ result: Result<Data?, RemoteError>) -> Void) {
```

Comment:
_Just checking, sorry if this is obvious, I don't have context around the code._

Seeing an `Optional` wrapped in a `Result` seems a bit odd to me. Is the case `.success(.none)`, that is a "successful" operation with no `Data` returned a legit state of the system? If it isn't, then we might want to use `Data` instead and return `.failure(<some error>)` if `Data` is `nil`.